libcloudforensics/providers/aws/internal/account.py

```diff
@@ -567,7 +567,10 @@ def ShareKMSKeyWithAWSAccount(self, kms_key_id, aws_account_id):
         'Action': [
             'kms:Encrypt',
             'kms:Decrypt',
-            'kms:ReEncrypt*'
+            'kms:ReEncrypt*',
+            # This permission is necessary to transfer encrypted EBS
```

Comment:
Please name the permission you're referring to explicitly.

Reply:
Added, also added a small blob for the perms above this one to make it clearer.